### PR TITLE
Create proof-of-concept-guide.md

### PR DESCRIPTION
**Pull request by GuyDuerFB**
Proof of concept guide for prospects.

### CODE REVIEW

**Comment by GuyDuerFB** [comment]
@fstr - Will appriciate your feedback. The relevant section is "Step 1 - Provide access to S3".

File: docs/general-reference/proof-of-concept-guide.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+# Firebolt Proof of Concept Technical Guidelines
+
+<aside>
+This guide should be used after you have met with your Firebolt designated sales team.
+* If you were trying to get to the Firebolt documentation homepage, please go to https://docs.firebolt.io
+* If you are looking to try out Firebolt and have yet to contact us please go to https://www.firebolt.io/getting-started-with-firebolt-workshop
+</aside>
+
+## Step 1 - Provide access to S3
+The data used in the POC should be made available in an S3 region (any region is fine). To ensure the Proof of Concept accurately reflects your production workload, it's recommended to have all or a significant portion of the data available.
```
Comment (fstr) [inline docs/general-reference/proof-of-concept-guide.md]: > any region is fine

Suggestion: Any region supported by Firebolt is fine with a link to the supported production regions in our documentaiton.

File: docs/general-reference/proof-of-concept-guide.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+<aside>
+Firebolt pays for data transfer costs if such occurs. You can find additional information in [this
+AWS guide](https://docs.aws.amazon.com/AmazonS3/latest/userguide/RequesterPaysBuckets.html). 
+</aside>
+
+Please go through the following steps to grant us access to the relevant S3 bucket. You can find additional information in [this AWS guide](https://docs.aws.amazon.com/AmazonS3/latest/userguide/add-bucket-policy.html)
+
+1. Connect to the AWS console and click on the relevant S3 bucket
+2. Go to *Permissions* and then scroll down and edit *Bucket Policy*
+3. Copy the following policy. make sure you replace `<bucket>` with the actual bucket name
```
Comment (fstr) [inline docs/general-reference/proof-of-concept-guide.md]:
```suggestion
3. Copy the following policy. Make sure you replace `<bucket>` with the actual bucket name
```

**Comment by fstr** [comment]
@GuyDuerFB The policy is looking good. Also the place where its added (bucket policy) is correct.

File: docs/general-reference/proof-of-concept-guide.md (hunk 1 above)
Comment (GuyDuerFB) [reply docs/general-reference/proof-of-concept-guide.md]: Thanks! Added.

**Review by molly-firebolt — APPROVED** [review]

**Pull request merged**